packages/bump/index.d.ts

```diff
@@ -14,8 +14,8 @@ declare module '@nivo/bump' {
     type SerieDerivedProp<Serie, T> = (serie: Serie) => T
 
     export interface BumpInputDatum {
-        x: string | number
-        y: string | number
+        x: string | number | null
+        y: string | number | null
```

Comment:
You mentioned `undefined` as a possible state, but I thought it might error. Just tested storybook locally and looks like it is possible. So one more suggestion I will apply.

```suggestion
        x?: string | number | null
        y?: string | number | null
```